components/log-viewer-webui/server/src/plugins/S3Manager.ts

```diff
@@ -63,15 +70,21 @@ class S3Manager {
  * Initializes a Fastify plugin, which decorates the application with an S3 manager at the
  * "s3Manager" property only when all plugin options are valid.
  */
-export default fastifyPlugin(async (app, options: {region: Nullable<string>}) => {
-    const {region} = options;
-    if (null === region) {
-        return;
-    }
+export default fastifyPlugin(
+    async (app, options: {region: Nullable<string>; profile: Nullable<string>}) => {
+        const {region, profile} = options;
+        if (null === region) {
+            return;
+        }
 
-    console.log(`Initializing S3Manager with region="${region}"...`);
-    app.decorate("s3Manager", new S3Manager(region));
-});
+        const logMessage = null === profile ?
+            `Initializing S3Manager with region="${region}"` :
+            `Initializing S3Manager with region="${region}" and profile="${profile}"...`;
+
+        console.log(logMessage);
```

Comment:
In my opinion, it's fine to log `profile=null` since only developers / service maintainers would read the logs. It's even easier for debugging when we always log `profile=xxx`.
```suggestion
        console.log(`Initializing S3Manager with region="${region}" and profile="${profile}"...`);
```